src/components/formio-wrapper.js

```diff
@@ -265,9 +267,11 @@ export class FormioWrapper {
         delete this.storedData._seenPages;
         this.wizard.page = this.storedData.page;
         delete this.storedData.page;
+
         this.wizard.data = this.storedData;
-        this.wizard.data[this.config.terms.dataName] = JSON.parse(termsStorage
-          .getItem(this.config.terms.termsStorageName));
+        this.wizard.data[this.config.terms.dataName] = JSON.parse(
```

Comment:
I was under the impression that we should be using wizard.submission.data as that is the 'true' underlying data which a subset of data is populated to wizard.data @ganapavz from your meetings with form.io dev's in usa is this the case?

Reply:
@duttonw, the suggestion from Form.io team was only to not manipulate wizard.data.